ga4gh/exceptions.py

```diff
@@ -0,0 +1,85 @@
+"""
+Module containing error objects we return to clients
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import ga4gh.protocol as protocol
+
+
+class ParentException(Exception):
+
+    def __init__(self):
+        super(ParentException, self).__init__()
+        self.message = ""
+        self.code = ""
+        self.description = ""
+        self.status = None
+
+    def toDict(self):
+        return {
+            "message": self.message,
+            "code": self.code,
+            "description": self.description,
+        }
+
+
+class BadRequestException(ParentException):
+
+    def __init__(self):
+        super(BadRequestException, self).__init__()
+        self.status = 400
```

Comment:
I think httpStatus would be a bit clearer here - these numbers might be a bit mysterious otherwise.